circuitClient.js

```diff
@@ -122,6 +124,68 @@ class CircuitClient {
       }]);
     });
   }
+  
+  async setMyStatusMessage(statusMessage) {
+    return this.client.setStatusMessage(statusMessage)
+    .catch(console.error);
+  }
+
+  getMyStatusMessage() {
+    return this.client.getStatusMessage()
+    .catch(console.error);
+  }
+
+  async setPresenceAvailable() {
+    const statusMsg = await this.client.getStatusMessage();
```

Comment:
typically you don't mix async with promise chaining, I think removing the .catch might be better or putting them in a try catch. According to the others google assistant has it's own error handling, you can probably remove the catches.